### PR TITLE
Add Unit Tests

### PR DESCRIPTION
**Pull request by daileytj**
Fixes PXBLUE-1926.

#### Changes proposed in this Pull Request:
- Add unit tests

### CODE REVIEW

**Comment by daileytj** [comment]
## Coverage Report (as of commit 9bb9aee)
Screen Shot 2021-06-17 at 10 59 18 AM

**Comment by jeffvg** [comment]
getting this with jest but can't find the specific version.....removed node modules but not able to run

image

**Comment by jeffvg** [comment]
could be coming out from @types/jest

File: login-workflow/setup-tests.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const originalConsoleError = console.error;
+console.error = (message) => {
+    // see: https://jestjs.io/docs/en/tutorial-react.html#snapshot-testing-with-mocks-enzyme-and-react-16
+    // see https://github.com/Root-App/react-native-mock-render/issues/6
+    if (message.startsWith('Warning:')) {
+        return;
+    }
```
Comment (joebochill) [inline login-workflow/setup-tests.js:33]: Not a huge fan of disabling warnings in the log, but we can revisit this if you log an issue @daileytj

**Review by joebochill — APPROVED** [review]

**Pull request merged**